cmd/lido-exporter/README.md

```diff
@@ -0,0 +1,53 @@
+# Lido Exporter
+
+Lido Exporter is a service that exports data from the Lido CSM smart contracts as Prometheus metrics.
+
+## Features
+
+- Collects metrics for penalties, exit requests, node operator info, keys info, bond info, and rewards info
+- Supports Holesky network (Mainnet support can be easily added)
+- Exports metrics in Prometheus format
+
+## Usage
+
```

Comment:
We should add here how to run it using the docker image published on DockerHub

Reply:
[Done](https://github.com/NethermindEth/sedge/pull/456/commits/466d65f5b91ba8357e450b3025905668920e7285)